framework/helpers/BaseHtml.php

```diff
@@ -953,9 +954,12 @@ public static function checkboxList($name, $selection = null, $items = [], $opti
      */
     public static function radioList($name, $selection = null, $items = [], $options = [])
     {
-        $encode = !isset($options['encode']) || $options['encode'];
-        $formatter = isset($options['item']) ? $options['item'] : null;
-        $itemOptions = isset($options['itemOptions']) ? $options['itemOptions'] : [];
+        $formatter = ArrayHelper::remove($options, 'item', null);
```

Comment:
no need to pass `null`, because 3rd argument of `ArrayHelper::remove` defaults to null